.github/workflows/latency-tracking.yml

```diff
@@ -1,13 +1,7 @@
-name: CI
+name: Latency Tracking
 
-# on:
-#  pull_request_target:
-# on:
-#   push:
-#     branches-ignore:
-#       - '**'  # temporally ignore all
 on:
-  workflow_dispatch:
+  - pull_request
```

Comment:
I think it was `pull_request_target` as we wanted the ability to _update_ a comment. Not sure if that's still required.

Reply:
I think `pull_request` will do the job while `pull_request_target` might be [dangerous](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target) in some cases.